include/RAJA/policy/atomic_builtin.hpp

```diff
@@ -190,17 +209,29 @@ RAJA_INLINE T atomicSub(builtin_atomic, T volatile *acc, T value)
 template <typename T>
 RAJA_INLINE T atomicMin(builtin_atomic, T volatile *acc, T value)
 {
-  return detail::builtin_atomic_CAS_oper(acc, [=](T a) {
-    return a < value ? a : value;
-  });
+  if (*acc < value)
```

Comment:
It would help readability here if 'return *acc' had curly braces around it.

Reply:
Perfectly fair, I'll tweak that momentarily.